quarkus-build-caching-extension/src/main/java/com/gradle/QuarkusBuildCache.java

```diff
@@ -218,7 +230,10 @@ private String getJarPackageType(Properties quarkusCurrentProperties) {
 
     private void configureInputs(MojoMetadataProvider.Context context, QuarkusExtensionConfiguration extensionConfiguration, Properties quarkusCurrentProperties) {
         context.inputs(inputs -> {
-            addOsInputs(inputs);
+            if (!isInContainerWithFixedImageBuild(quarkusCurrentProperties, false)) {
```

Comment:
just to confirm, we still need this for the non-container strategy because it affects the outcome?
why doesn't it affect the outcome on the in container strategy?

Reply:
Yes to your first question.
Native executable compilation depends on system libraries, we are assuming that building inside the same image would provide the same system libraries.